_posts/2022-09-13-#1689.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: pr
+date: 2022-09-13
+title: "Expose new BumpChannelClose event for channels with anchor outputs"
+authors: ["wpaulino"]
+pr: 1689
+components: ["tests"]
```

Comment:
I think we just need to change this from “tests” to the module/area of code we’re working in. I think this was left from one of the other copy pasta for the other one. Oops! 😅

Reply:
Yeah sure, changed to "chain monitoring", though let me know if better suggestion!